src/seedsigner/views/tools_views.py

```diff
@@ -236,6 +240,58 @@ def run(self):
 
 
 
+"""****************************************************************************
+    Coin flips Views
+****************************************************************************"""
+class ToolsCoinEntropyMnemonicLengthView(View):
+    def run(self):
+        TWELVE = f"12 words ({mnemonic_generation.COIN__NUM_FLIPS__12WORD} flips)"
+        TWENTY_FOUR = f"24 words ({mnemonic_generation.COIN__NUM_FLIPS__24WORD} flips)"
```

Comment:
Moving these constants to be class-level attrs will allow us to specify 12 vs 24 in various FlowTest scenarios.